images/yamllint/Dockerfile

```diff
@@ -0,0 +1,2 @@
+FROM python:3.13.0a3-bookworm
```

Comment:
```suggestion
FROM python:3-bookworm
```

Reply:
@mkumatag I have changed and pushed a new image.